cmd/meroxa/builder/builder.go

```diff
@@ -679,14 +675,30 @@ func buildCommandWithFeatureFlag(cmd *cobra.Command, c Command) {
 			}
 		}
 
-		flagRequired := v.FeatureFlag()
+		flagRequired, err := v.FeatureFlag()
 		userFeatureFlags := global.Config.GetStringSlice(global.UserFeatureFlagsEnv)
 
```

Comment:
small recommendation to mage the error closer to where it is being used
```suggestion
		userFeatureFlags := global.Config.GetStringSlice(global.UserFeatureFlagsEnv)
		flagRequired, err := v.FeatureFlag()

```